src/WordPress.php

```diff
@@ -1439,7 +1439,11 @@ protected function update_ssl( $assoc_args ) {
 
 		$extra_flags = '--precise';
 		$extra_flags .= ( 'wp' === $this->site_data['app_sub_type'] ) ? '' : ' --network';
-		EE::exec( sprintf( \EE_DOCKER::docker_compose_with_custom() . ' exec php wp search-replace http://%1$s https://%1$s %2$s', $this->site_data['site_url'], $extra_flags ), true, true );
+		$ssl = get_flag_value( $assoc_args, 'ssl' );
+		$from = $ssl === 'on' ? 'http' : 'https';
```

Comment:
Please run phpcbf

Reply:
I've run it, found no issues, have not run phpstorm formatter as I don't have config, and importing our shared config overrides my other settings such as theme etc... IMO let's find a formatting solution that is FOSS and can be run from the command line. Then we can maybe run it in CI itself. Till then, let's keep the code as-is. 

One of the reasons for not using phpstorm formatter is also that phpcbf and phpstorm formatter were giving conflicting suggestions for formatting, which can also be confusing.